### ISSUE TITLE
Add .gitignore 

### ISSUE THREAD

**Issue by chudovsky** [Lunnox/RCP_Test#2]
.gitignore must be placed in root folder and contain, ex.

```
# Compiled class file
*.class

# Log file
*.log

# BlueJ files
*.ctxt

# Mobile Tools for Java (J2ME)
.mtj.tmp/

# Package Files #
*.jar
*.war
*.nar
*.ear
*.zip
*.tar.gz
*.rar

# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
hs_err_pid*
bundles/com.chudovsky.tycho.plugin1/target
features/com.chudovsky.tycho.feature/target
releng/com.chudovsky.tycho.update/target
bundles/com.chudovsky.tycho.plugin1/target
releng/com.chudovsky.tycho.product/target
bundles/com.chudovsky.tycho.rcp/target
.vscode
.settings
/.metadata/
*.project
test/com.chudovsky.tycho.rcp.it.tests/target
test/com.chudovsky.tycho.rcp.tests/target
```

**Comment by chudovsky** [comment]
After adding .gitignore, please remove commited binary files

**Comment by Lunnox** [comment]
closed with branch fix/issue#2
